pandas/tests/io/test_packers.py

```diff
@@ -317,6 +318,19 @@ def test_timedeltas(self):
             i_rec = self.encode_decode(i)
             assert i == i_rec
 
+    def test_periods(self):
+        # 13463
```

Comment:
can you test period index & interval index explicity. DId you say timedeltaindex is fixed as well? if not can you add a test for that.

Reply:
IntervalIndex and TimedeltaIndex are now tested in TestIndex along with all the other index subclasses (see addition at lines 352-353). Would it be better to leave that way so all the Index subclasses are in one place and treated the same way rather than separating 352-353 into its own test? E.g. it was because I didn't see TimedeltaIndex in the dict in TestIndex that I added it (and, in turn, found out that TimedeltaIndex was broken).

PeriodIndex worked before (and was tested, see line 334) but the Period object itself couldn't be serialized (#13463). That's being explicitly tested here at line 321 (serialization of Period and Interval).

Reply:
oh ok that's fine then.